frontend/components/dashboard/notifications/NotificationsManagementModal.vue

```diff
@@ -0,0 +1,87 @@
+<script lang="ts" setup>
+import {
+  faCog,
+  faGaugeSimpleMax,
+  faMonitorWaveform,
+  faBolt,
+  faNetworkWired
+} from '@fortawesome/pro-solid-svg-icons'
+
+const { t: $t } = useI18n()
+
+const visible = defineModel<boolean>()
+</script>
+
+<template>
+  <BcDialog
+    v-model="visible"
+    :header="$t('dashboard.notifications.management.title')"
+    class="notifications-management-modal-container"
+  >
+    <TabView lazy class="notifications-management-tab-view">
+      <TabPanel>
+        <template #header>
+          <BcTabHeader :header="$t('dashboard.notifications.management.tabs.general')" :icon="faCog" />
+        </template>
+        General coming soon!
+      </TabPanel>
+      <TabPanel>
+        <template #header>
+          <BcTabHeader :header="$t('dashboard.notifications.management.tabs.dashboards')" :icon="faGaugeSimpleMax" />
+        </template>
+        Dashboards coming soon!
+      </TabPanel>
+      <TabPanel>
```

Comment:
Maybe disable the tabs that are very likely not come in the near future with the development flag

Reply:
Done.